include/lattices/lww_pair_lattice.hpp

```diff
@@ -52,7 +52,7 @@ class LWWPairLattice : public Lattice<TimestampValuePair<T>> {
   }
 
  public:
-  LWWPairLattice() : Lattice<TimestampValuePair<T>>() {}
+  LWWPairLattice() : Lattice<TimestampValuePair<T>>(TimestampValuePair<T>()) {}
```

Comment:
why modify this?

Reply:
I had to take out the default (zeroarg) constructor for Lattice, which was making bottom lattices in a nongeneralizable way. Here, I'm replacing the call to the default constructor by providing an explicit bottom value. Same for your other comment as well.

Reply:
got it